### PR TITLE
enh(ws): add api commit offset

### PR DESCRIPTION
**Pull request by AdamEECS**

### CODE REVIEW

File: taos-ws-py/tests/test_commit_offset.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        "create table `table` (ts timestamp, v int)",
+        "alter table stb1 add column new1 bool",
+        "alter table stb1 add column new2 tinyint",
+        "alter table stb1 add column new10 nchar(16)",
+        "alter table stb1 modify column new10 nchar(32)",
+        "alter table stb1 drop column new10",
+        "alter table stb1 drop column new2",
+        "alter table stb1 drop column new1",
+        "alter table `stb2` add tag new1 bool",
+        "alter table `stb2` rename tag new1 new1_new",
```
Comment (sheyanjie-qq) [inline taos-ws-py/tests/test_commit_offset.py:45]: alter table is not needed in commit tests

File: taos-ws-py/tests/test_commit_offset.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    while 1:
+        message = consumer.poll(timeout=1.0)
+        if message:
+            id = message.vgroup()
+            topic = message.topic()
+            database = message.database()
+            print(f"vgroup: {id}, topic: {topic}, database: {database}")
+
+            committed = consumer.committed(topic, id)
+            print(f"committed: {committed}")
```
Comment (sheyanjie-qq) [inline taos-ws-py/tests/test_commit_offset.py:83]: in unit tests, we use assert to check test success or failed

File: tests/test_tmq.py [hunk 3]
```diff
@@ -163,10 +178,30 @@ def test_tmq_assignment():
                     f"insert into t{i} using stb1 tags(true, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, '1', '1') values (now, true,1,1,1,1,1,1,1,1,1,1,1,'1','1')"
                 )
 
-        consumer.poll(5)
+        message = consumer.poll(5)
```
Comment (sheyanjie-qq) [inline tests/test_tmq.py:181]: same as test_tmq.py 156-171

File: taos-ws-py/tests/test_commit_offset.py (hunk 1 above)
Comment (AdamEECS) [reply taos-ws-py/tests/test_commit_offset.py]: done

File: taos-ws-py/tests/test_commit_offset.py (hunk 2 above)
Comment (AdamEECS) [reply taos-ws-py/tests/test_commit_offset.py]: done

File: tests/test_tmq.py (hunk 3 above)
Comment (AdamEECS) [reply tests/test_tmq.py:181]: This is different from lines 156-171. The purpose of lines 156-171 is to subscribe to earliest data, and then lines 173-179 insert new data, which will be subscribed here.

**Pull request merged**